data/hai/hai reveal 5 philosophers.txt

```diff
@@ -448,39 +455,39 @@ event "devil-run smugglers visible"
 
 	# Route to Almaaz (and nearby systems)
 	system "Rigel"
-		add fleet "Smuggler (Hai Traffickers)" 5000
+		add fleet "Smuggler (Hai Traffickers)" 4000
 	system "Alnitak"
-		add fleet "Smuggler (Hai Traffickers)" 7000
+		add fleet "Smuggler (Hai Traffickers)" 6000
 	system "Saiph"
-		add fleet "Smuggler (Hai Traffickers)" 10000
+		add fleet "Smuggler (Hai Traffickers)" 8000
 	system "Betelgeuse"
-		add fleet "Smuggler (Hai Traffickers)" 10000
+		add fleet "Smuggler (Hai Traffickers)" 8000
 	system "Mintaka"
-		add fleet "Smuggler (Hai Traffickers)" 5000
+		add fleet "Smuggler (Hai Traffickers)" 4000
 	system "Gorvi"
-		add fleet "Smuggler (Hai Traffickers)" 5000
+		add fleet "Smuggler (Hai Traffickers)" 4000
 	system "Tortor"
-		add fleet "Smuggler (Hai Traffickers)" 5000
+		add fleet "Smuggler (Hai Traffickers)" 4000
 	system "Unagi"
-		add fleet "Smuggler (Hai Traffickers)" 7000
+		add fleet "Smuggler (Hai Traffickers)" 6000
 	system "Hatysa"
-		add fleet "Smuggler (Hai Traffickers)" 5000
+		add fleet "Smuggler (Hai Traffickers)" 4000
 	
 	# Route to Almach
 	system "Zaurak"
-		add fleet "Smuggler (Hai Traffickers)" 7000
+		add fleet "Smuggler (Hai Traffickers)" 5500
 	system "Sheratan"
-		add fleet "Smuggler (Hai Traffickers)" 7000
+		add fleet "Smuggler (Hai Traffickers)" 5500
 	system "Ruchbah"
-		add fleet "Smuggler (Hai Traffickers)" 7000
+		add fleet "Smuggler (Hai Traffickers)" 5500
 	system "Alpheratz"
-		add fleet "Smuggler (Hai Traffickers)" 7000
+		add fleet "Smuggler (Hai Traffickers)" 5500
 	system "Achernar"
-		add fleet "Smuggler (Hai Traffickers)" 7000
+		add fleet "Smuggler (Hai Traffickers)" 5500
 	system "Al Dhanab"
-		add fleet "Smuggler (Hai Traffickers)" 7000
+		add fleet "Smuggler (Hai Traffickers)" 5500
 	system "Mirach"
-		add fleet "Smuggler (Hai Traffickers)" 7000
+		add fleet "Smuggler (Hai Traffickers)" 5500
```

Comment:
Same deal, if you've had to end up on this plot line, then you're meant to have to _work_ for the information you find. It should not be easy.

Reply:
In retrospect, the numbers aren't changed enough to matter anyway. Will revert.